### PR TITLE
# ember-cli Meeting Notes - 2017-06-29

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: ember-cli/2017-06/june-29.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-06-29
+
+## Attendees
+
+
+## Agenda
```
Comment (kellyselden) [inline ember-cli/2017-06/june-29.md]: https://github.com/ember-cli/rfcs/pull/105 Use optionalDependencies for addon resolution.

Comment (hjdivad) [inline ember-cli/2017-06/june-29.md]: using [prettier](https://github.com/prettier/prettier) on the ember-cli codebase.

Comment (rwjblue) [inline ember-cli/2017-06/june-29.md]: 💅

Comment (rwjblue) [inline ember-cli/2017-06/june-29.md]: discuss / push forward https://github.com/ember-cli/rfcs/pull/100

Comment (rwjblue) [inline ember-cli/2017-06/june-29.md]: parallel broccoli-persistent-filter

**Comment by hjdivad** [comment]
@kellyselden can we merge this?

**Comment by kellyselden** [comment]
@hjdivad Yep. I don't have permission though.

**Comment by hjdivad** [comment]
@kellyselden well that's the boat i'm in ^_^

**Comment by hjdivad** [comment]
@rwjblue @stefanpenner +r?

**Pull request merged**